playerctl/playerctl-cli.c

```diff
@@ -16,279 +16,359 @@
  * along with playerctl If not, see <http://www.gnu.org/licenses/>.
  *
  * Copyright © 2014, Tony Crisci
+ * Copyright © 2016, Jente Hidskes <hjdskes@gmail.com>
```

Comment:
Where did you read this? I maintain the copyright to the code I wrote, so there should be a line in there to make this explicit. However, I'm fine with changing this line to `Copyright © 2014, 2016, Tony Crisci and contributors`, in combination with a `CONTRIBUTORS` file in the project root.

Perhaps [this](http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/) is an interesting read. I'm interested in hearing more on this, I haven't heard of it before.